README.md

```diff
@@ -83,27 +83,39 @@ The e2e tests are divided into u/s (upstream) basic workflows on [assisted-test-
 
 ### Subsystem tests pre-configuration
 
-* Run minikube on your system
-* Enable [registry addon](https://minikube.sigs.k8s.io/docs/handbook/registry/) on your minikube
+* Run [minikube](https://minikube.sigs.k8s.io/docs/start/) on your system
+* Enable [registry addon](https://minikube.sigs.k8s.io/docs/handbook/registry/) on your minikube:
+```
+minikube addons enable registry
+```
 * Set `LOCAL_ASSISTED_ORG` to point to your local registry address
-
 ```bash
 export LOCAL_ASSISTED_ORG=localhost:5000
 ```
-
+* Redirect the local registry to the minikube registry:
+```bash
+docker run --rm -it --network=host alpine /bin/ash -c "apk add socat && socat TCP-LISTEN:5000,reuseaddr,fork TCP:$(minikube ip):5000"
```

Comment:
why? you can do that with xinet or just kubectl port-forward

Reply:
I was just following the documentation of the registry addon, I will change it to kubectl port-forward

Reply:
please don't forget to forward the output no null